### ISSUE TITLE
UPDATE: Recent influx of support

### ISSUE THREAD

**Issue by drwhut** [drwhut/tabletop-club#147]
***TL;DR: Expect it to be some time before I reply to newly created issues + PRs.***

Hey everyone! I just wanted to give a quick update on where my mind is at currently.

Firstly, I just wanted to say thank you to everyone who has taken an interest in the project! At one point I had a look at the number of stars the GitHub project had, and I swear I remember the number being around 80, then I blinked and the number went to over 600!! Thank you all again for the support, it means a lot to me.

However, over the past couple of days, I've noticed that I haven't really touched the code at all, and that I've pretty much *only* been responding to issues and reviewing pull requests - while the project has been open source since the initial commit, I have never had this many issues/PRs created in such a short time span. Internally, I was not prepared for the rate at which the project would gain traction, and it has unfortunately taken somewhat of a toll on me.

Thankfully I noticed the warning signs early, and the last thing I want to do is burn out the moment the project starts getting attention. So I've made this post to basically say that I'll be adjusting the way I handle issues on GitHub, which is to focus on a few at a time, rather than force myself to reply to every notification I get. Hopefully this way, I'll be able to discuss/review each issue with the attention it deserves, and my mental health should be able to cope with the influx better.

Bear in mind, this is not a post saying "please stop creating issues and pull requests", this is more of a warning that it may take some time for me to get around to replying.

Thank you for your understanding,
Ben

*P.S. If an issue is urgent, please say so in the title and I'll get round to it as soon as possible.*
